sandbox-tools/merge-repos/src/config.ts

```diff
@@ -81,14 +102,162 @@ export const reposToSyncAndMerge: IRepoSyncDetails = {
     }
 };
 
+/**
+ * Identifies the web packages that should be merged from the staging branch to main and the 
+ * location that the package should be moved to.
+ */
+export const foldersToMerge: IMergePackageDetail[] = [
+    { name: "@opentelemetry/api", destPath: "pkgs/api/", srcPath: "auto-merge/js/api", bundleName: "otel-sndbx.api", bundleNamespace: "opentelemetry.sandbox.web.api" }
```

Comment:
This defines the packages to merge and relocate from the auto-merge branch (which is essentially the js and contrib repositories in their entirety)

name -> The expected package name (used to validate that the config is reading/moving the right folder
destPath -> The location to move it to
srcPath -> The location to move it from (essentially `git mv srcPath destPath` it's a little more complicated to get all the files)
bundleName -> The filename of the *.js file for the rollup generated bundles
bundleNamespave -> The namespace (global) to use for the bundle
noTests -> The package doesn't have any tests
noWorkerTests -> This package doesn't currently support the automatic creation of web worker tests

Reply:
The API mapping does not have to be here anymore since it is now in the JS repo?

Reply:
it is still its own package. I think every package needs an entry eventually

Reply:
Correct, which is what this `foldersToMerge` configuration is about. All packages that can be run on the web will eventually be in this list. And when they move from experimental to packages (in the core repository's) then we will need to update this configuration.